flycheck.el

```diff
@@ -6558,6 +6561,56 @@ See URL `https://docs.python.org/3.4/library/py_compile.html'."
           line ", " column ", " (one-or-more not-newline) line-end))
   :modes python-mode)
 
+(flycheck-define-checker r-lint
```

Comment:
Do we really need this syntax checker?  It seems as if it's not maintained anymore (last release almost two years ago, no public repo).  To me it looks as if `lintr` is the way to go.